### ISSUE TITLE
Could not load type 'Stripe.SubscriptionSharedOptions' from assembly 'Stripe.net, Version=37.6.0.0

### ISSUE THREAD

**Issue by MiroJ** [stripe/stripe-dotnet#2060]
**Background:**
I have code, which works with version 34.6.0.0 but stopped working when I updated to 37.4.0.0, and even with the latest 37.6.0.0. The error message is "Could not load type 'Stripe.SubscriptionSharedOptions' from assembly 'Stripe.net, Version=37.6.0.0..."
The issue started between 34.6.0.0 and 34.26.0.0 as I tested with the latter and it breaks the same way.

**Implementation:**
I have a class that I manage all Stripe functionality - StripeService. In its constructor, I set my ApiKey:
`StripeConfiguration.ApiKey = ConfigurationManager.AppSettings["STRIPE_API_SECRET"];`
My project is ASP.NET 4.6.1. 

**Problem:**
The class has two `async` methods. In a single method of another web controller, I instantiate my StripeService class. After that, I call first method, then the second one - both with `await`. The first completes successfully, the second fails at the call to it. It does not even get inside the method, where on its first line I have a breakpoint. The IDE stops at the break-point when I run version 34.6.0.0.

**Comment by remi-stripe** [comment]
@MiroJ The class `SubscriptionSharedOptions` was removed in 34.10.1 in this PR: https://github.com/stripe/stripe-dotnet/releases/tag/v34.10.1 It was an internal class that we used to share parameters between the Create and Update APIs for Subscriptions.

Is it possible your code is still referencing or using it incorrectly? If so, you'd need to stop using this and just directly set the information on `SubscriptionCreateOptions` or `SubscriptionUpdateOptions`.
